_tasks/.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "plugins": ["@babel/plugin-syntax-top-level-await"],
+  "presets": [
```

Comment:
i wonder if this could be made to `extends` the parent babelrc rather than replacing it?

Reply:
Yeah, that would be preferable. I'll look into it.